card/mod/search/set/type/search_type.rb

```diff
@@ -20,20 +20,8 @@ def search_result_view
 end
 
 format :json do
-  view :export do |args|
-    # avoid running the search from options and structure that
-    # case a huge result or error
```

Comment:
no longer a problem?

Reply:
hmm, I must have gotten confused going back and forth between this and the wikirate version.  See https://github.com/wikirate/wikirate/pull/791/files#diff-89b57322dad210a7070c1ab0ddd00c9f

I'll move that code here (and get rid of the wikirate file altogether)